ddpui/tests/core/ai/test_layer2_integration.py

```
@@ -0,0 +1,469 @@
+from ddpui.models.visualization import Chart
+
+
+class TestLayer2Integration(TestCase):
+    """Integration tests for Layer 2 components"""
+
+    def setUp(self):
+        """Set up test fixtures"""
+        # Create test org
+        self.org = Org.objects.create(name="Test Education Org", slug="test-edu-org")
+        self.user = User.objects.create_user(
+            username="testuser", email="testuser@example.com", password="password"
```

Ruff (0.14.13)

Possible hardcoded password assigned to argument: "password"

(S106)